proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_sv_SE.properties

```diff
@@ -0,0 +1,67 @@
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Du är redan ansluten till denna servern!
+velocity.error.already-connected-proxy=Du är redan ansluten till denna proxyservern!
+velocity.error.already-connecting=You are already trying to connect to a server!
+velocity.error.already-connecting=Du ansluter redan till denna servern!
+velocity.error.cant-connect=Kan inte ansluta till {0}: {1}
+velocity.error.connecting-server-error=Kan inte ansluta till {0}. Vänligen försök igen senare.
+velocity.error.connected-server-error=Ett fel med din anslutning till {0} inträffade.
+velocity.error.internal-server-connection-error=Ett internt serverfel inträffade.
+velocity.error.logging-in-too-fast=Du anslutar dig för snabbt, försök igen senare.
+# Perhaps replace "klient" with "spel"? I feel klient is a little too technical/unusual in normal speech.
+# Kanske vi borde ersätta "klient" med "spel"? Det känns som klient är lite för tekniskt/onaturligt i vardagligt tal.
+velocity.error.online-mode-only=Du är inte inloggad på ditt Minecraft konto. Om du är inloggad på ditt konto, försök att start om din Minecraft klient.
+velocity.error.player-connection-error=Ett fel med din anslutning inträffade.
+velocity.error.modern-forwarding-needs-new-client=Den här servern stöder endast Minecraft 1.13 och nyare.
+velocity.error.modern-forwarding-failed=Din server skickade inte någon vidarebefordran till proxyservern. Se till att servern är konfigurerad för Velocitybefordran.
+velocity.error.moved-to-new-server=Du blev sparkat ut ur {0}: {1}
+velocity.error.no-available-servers=Det finns inte någon tillgängliga servrar att ansluta dig till. Försök igen senare eller kontakta en administratör.
+
+# Commands
+velocity.command.generic-error=Ett fel med kommandot inträffade.
+velocity.command.command-does-not-exist=Detta kommandot finns inte.
+
+velocity.command.players-only=Endast spelare får köra det här kommandot.
+velocity.command.server-does-not-exist=The specified server {0} does not exist.
+# Perhaps/kanske: Servern {0} finns inte.
+velocity.command.server-does-not-exist=Den angivna servern {0} finns inte.
+
+velocity.command.server-current-server=Du är för tilfället ansluten till {0}.
+velocity.command.server-too-many=För många servrar är sätts upp. Använd tabslutförande för att se alla tillgängliga servrar.
+velocity.command.server-available=Tillgängliga servrar:
+velocity.command.server-tooltip-player-online={0} spelare ansluten
+velocity.command.server-tooltip-players-online={0} spelare ansluten
+velocity.command.server-tooltip-current-server=För tilfället ansluten till denna servern
```

Comment:
"Du är..." in front? Sounds a bit weird just floating like that, doesn't really work the same as in English